ui/app/src/components/GlobalConfigManagement/GlobalConfigManagement.tsx

```diff
@@ -122,8 +126,18 @@ export function GlobalConfigManagement() {
           }
         })
       );
+    } else if (contentSize > MAX_CONFIG_SIZE) {
+      dispatch(
+        showSystemNotification({
+          message: formatMessage({
+            defaultMessage: 'Maximum configuration size exceeded, please reduce it in order to properly save.'
+          }),
+          options: {
+            variant: 'error'
+          }
+        })
+      );
```

Comment:
With the [new approach](https://github.com/craftersoftware/craftercms/issues/617#issuecomment-1813948305) in mind, perhaps we simply disable the Save button and we don't need this.